cmd/cli/main.go

```diff
@@ -32,6 +32,11 @@ func main() {
 	var rootCmd = &cobra.Command{
 		Use: "mainflux-cli",
 		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			err := cli.ParseConfig()
```

Comment:
If error is not needed below, and it looks like it is not, you can embed this check and function call in one line.

Reply:
Sure, should I do the same in `getUserAuthToken`?

Reply:
Not sure that you can do the same here, because you use value below.